network/dropper.go

```diff
@@ -54,9 +54,12 @@ func handleDropping(log zerolog.Logger, wg *sync.WaitGroup, cfg *Config, peers p
 		addresses := peers.Addresses()
 		address := addresses[rand.Int()%len(addresses)]
 		err := peers.Drop(address)
+
 		if err != nil {
 			log.Error().Err(err).Msg("could not drop peer")
 			continue
+		} else {
```

Comment:
There is no need for an else statement. The if already has a `continue`.

Reply:
Good point, fixed.